src/modules/filters/sort.ts

```diff
@@ -0,0 +1,33 @@
+import { onSortClick } from '../interfaces/customTypes';
+
+export class Sort {
```

Comment:
I would avoid to name anything with just pure word "Sort". Why it's not a keyword, it's confusing to have general "Sort". I would suggest to add some specifics.

Reply:
Done.